### ISSUE TITLE
FR: Allow display of both placeholder and fallback

### ISSUE THREAD

**Issue by choumx** [ampproject/amphtml#22704]
### Status quo

```
Display the placeholder while the content is loading.
If the content loads successfully, hide the placeholder and display the content.
If the content fails to load:
    If there's a fallback element, display the fallback.
    Otherwise, continue displaying the placeholder.
```
https://amp.dev/documentation/guides-and-tutorials/develop/style_and_layout/placeholders#interaction-of-placeholders-and-fallbacks

### Proposal

In some cases, it would be useful to display both the placeholder and fallback. 

For example:
- amp-list's placeholder contains stale content
- The amp-list fails to fetch fresh content due to network error
- Placeholder should still be displayed
- [New] Show an error message (fallback) to inform the user that content is stale

### Syntax

```html
<!-- Shows both fallback and placeholder on XHR failure. -->
<amp-list fallback="keep-placeholder">
  <div fallback>Failed to fetch content</div>
  <div placeholder>...</div>
</amp-list>
```

/to @ampproject/wg-ui-and-a11y

**Comment by aghassemi** [comment]
@choumx I assume you have more context on who/what needs this. What is priority in your opinion?

**Comment by choumx** [comment]
Closing in favor of a feature specific to amp-list: #23465
